src/core/services/LineFactoryService.ts

```diff
@@ -7,7 +7,7 @@ import { TransactionResponse } from '../types';
 
 import { LineFactoryABI } from './contracts';
 
-const { Arbiter_GOERLI, SwapTarget_GOERLI, LineFactory_GOERLI, KibaSero_oracle } = getConfig();
+const { LineFactory_GOERLI } = getConfig();
```

Comment:
should be the new capitalized name. There will be other places in this file too.